main_test.go

```diff
@@ -275,39 +275,39 @@ func TestWithEnvVars(t *testing.T) {
 			name:       "Set OPKSSH_DEFAULT to bad value",
 			envVar:     "OPKSSH_DEFAULT",
 			envValue:   "badvalue",
-			args:       []string{"opkssh", "login"},
+			args:       []string{"opkssh", "login", "--config-path", "/foo/bar"},
```

Comment:
Why is this test changed?

Reply:
IIUC correctly if it finds ~/.opk/config.yml it will fail. So I intentionally override config path to not collide test results with expected ones.